state_machine/applications/flight/lib/logs.py

```diff
@@ -61,7 +95,7 @@ def unpack_beacon(bytes):
      vbatt, cpu_temp, imu_temp,
      gyro0, gyro1, gyro2,
      mag0, mag1, mag2,
-     rssi, fei) = struct.unpack(beacon_format, bytes)
+     rssi, fei,) = struct.unpack(beacon_format, bytes)
```

Comment:
Is there a reason this comma was added?

Reply:
No, must have been a remnant of a previous change i had worked on, sorry!